src/userdel.c

```diff
@@ -1152,7 +1153,7 @@ int main (int argc, char **argv)
 	update_groups ();
 
 	if (rflg) {
-		errors += remove_mailbox ();
+		errors = remove_mailbox () || errors;
```

Comment:
Should we do `|=`?  It should work as good as a hypothetical `||=`, I guess.

Reply:
I'd prefer to stay with boolean operations here just to clarify what's going on. Heh, it helped with finding the logic bug I introduced, right?

Reply:
On the one hand yes, it helped see the bug.  I honestly don't know if I would have catched it as `&=`.

On the other hand, if you had written `errors = errors || remove_mailbox();` I wouldn't probably have noticed that we're short-cutting the call, and so `|=` might have been safer in that regard.

I don't know what to do.  What do you think?  I guess I'll take whatever you prefer, since you're the author.

Reply:
Cc: @hallyn

Reply:
Hmmm, sounds good.  Thanks!